tests/Documents/GetRecordDocumentTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Terraformers\OpenArchive\Tests\Documents;
+
+use SilverStripe\Assets\File;
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\Dev\SapphireTest;
+use Terraformers\OpenArchive\Documents\GetRecordDocument;
+use Terraformers\OpenArchive\Extensions\VersionedOaiRecordManager;
+use Terraformers\OpenArchive\Formatters\OaiDcFormatter;
+use Terraformers\OpenArchive\Jobs\OaiRecordUpdateJob;
+use Terraformers\OpenArchive\Tests\Mocks\FileExtension;
+use Terraformers\OpenArchive\Tests\Mocks\SiteTreeExtension;
+
+class GetRecordDocumentTest extends SapphireTest
+{
+
+    protected static $fixture_file = 'GetRecordDocumentTest.yml'; // phpcs:ignore
+
+    protected static $required_extensions = [
+        SiteTree::class => [
+            VersionedOaiRecordManager::class,
+            SiteTreeExtension::class,
+        ],
+    ];
+
+    public function testProcessOaiRecord(): void
+    {
+        // Mock OaiDcFormatter and mock GetRecordDocument
+        $formatter = new OaiDcFormatter();
+        $xmlDocument = GetRecordDocument::create($formatter);
+
+        // Get a mock OaiRecord and Page record
+        $page = $this->objFromFixture(SiteTree::class, 'page');
+        $oaiRecord = $page->OaiRecords()->first();
+
+        // Get the contents of the document
+        $originalDocumentBody = $xmlDocument->getDocumentBody();
+
+        // Confirm that this record has the contents of the mock file
+        $this->assertNotNull($originalDocumentBody);
+        $this->assertStringNotContainsString('<header>', $originalDocumentBody);
+        $this->assertStringContainsString('OAI-PMH', $originalDocumentBody);
+        $this->assertStringContainsString('<responseDate id="responseDate">', $originalDocumentBody);
+        $this->assertStringContainsString(
+            '<request id="request" verb="GetRecord" metadataPrefix="oai_dc"/>',
+            $originalDocumentBody
+        );
+        $this->assertEquals('0', $oaiRecord->Deleted);
+
+        // Run the test function with our mock Oai record
+        $xmlDocument->processOaiRecord($oaiRecord);
+
+        // Extract the document body after the main test has been executed
+        $extractedDocumentBody = $xmlDocument->getDocumentBody();
+
+        // Assert that we have the correct header attributes and that we do not see a 'status' element
+        $this->assertStringContainsString('<header>', $extractedDocumentBody);
+        $this->assertStringContainsString('<identifier>', $extractedDocumentBody);
+        $this->assertStringContainsString('<datestamp>', $extractedDocumentBody);
+        $this->assertStringNotContainsString("<status>deleted</status>", $extractedDocumentBody);
```

Comment:
Update to check that our header (with status) is not present.
```suggestion
$this->assertStringNotContainsString('<header status="deleted">', $processedDocumentBody);
```